.config/systemd/user/run-and-send-to-graphite:half-hourly@.timer

```diff
@@ -5,5 +5,8 @@ Description=run run-and-send-to-graphite@%i each 30 minutes
 # OnUnitActiveSec is broken: https://github.com/systemd/systemd/issues/6680
 OnCalendar=*:00,30
 
+# randomize start to avoid using too much ressources
+RandomizedDelaySec=10m
```

Comment:
As `AccuracySec` defaults to `1m`, this will run multiple timers with a high probability. So I would also put `AccuracySec` to `1s` or so.

Reply:
ho right, this leaves 10 slots to do that; hum, as the cas login process takes around 30s (and indeed is very ressource intensive), we can lower it to that without much change of cross-run. it would give 20 slots for 2 "heavy" services and 3 cothority-related lighter service.

or we can say that it will unlikely happen that both get allocated close timerange, and set it to `1s` (that a not-so-frequent wakeup in my mind)

Reply:
I would put `1s` here and `1us` for the minutely one.